_posts/2017/2017-05-09-how-to-successfully-run-a-tech-blog.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "How to successfully run a company tech blog"
+date: "2017-05-09 10:00:00"
+image: blog-header/social-like.jpg
+categories: tech-stories
+authors: ["Rose"]
+---
+
+Since we develop e-commerce solutions at [ePages](https://www.epages.com/us/), there were enough techies willing to help us out when we first decided to set up a [company tech blog](https://developer.epages.com/blog) in spring 2015.
+Our [API documentation](https://developer.epages.com/apps) was already [Jekyll](https://jekyllrb.com/)-based, so it came natural to connect our new blog to it.
+The new Social Media adventure, including the [blog](https://developer.epages.com/blog) as well as a [Twitter account](https://twitter.com/epagesdevs), started two years ago.
+
+In the following I would like to give you some deeper insights on what we have learnt ever since we began our blog journey in order to generate relevant content for our readers and how we engaged in the developer community via Twitter.
```

Comment:
In the following, I will give you some detailed insights on what we have learnt ever since we began our social media journey and how we engaged in the developer community with our tech blog and Twitter.